.github/workflows/main.yml

```diff
@@ -21,6 +21,8 @@ jobs:
             config: --config=ci
 
     steps:
+      - name: install and run cargo-sort
+        run:  cargo install cargo-sort && cargo-sort -w
```

Comment:
Move this down to the bottom, in the `cargo` job, after the `cargo fmt --all -- --check`